lib/WP_Auth0_LoginManager.php

```diff
@@ -355,15 +359,16 @@ public function implicit_login() {
 	 * Attempts to log the user in and create a new user, if possible/needed.
 	 *
 	 * @param object $userinfo - Auth0 profile of the user.
-	 * @param string $id_token - user's ID token returned from Auth0.
-	 * @param string $access_token - user's access token returned from Auth0; not provided when using implicit_login().
+	 * @param null|string $id_token - user's ID token returned from Auth0.
+	 * @param null|string $access_token - user's access token returned from Auth0; not provided when using implicit.
```

Comment:
> not provided when using implicit.

how not?

Reply:
We don't request one and don't need one. We're not pinging back to Auth0 in this case, which is the purpose of implicit, the WP server cannot connect to auth0.com for whatever reason.

Reply:
I get it that implicit doesn't receive an access_token since you're not requesting it. Fine. But This method doesn't know if the user called implicit flow or code flow. So, the description of the method should just say that null values are accepted for access_token and refresh_token, just that. And if you want, in the method that handles the implicit result parsing you can add in the docblock that you'll never receive an access_token or refresh_token there.